core/tests/puppeteer-acceptance-tests/functions/is-element-clickable.ts

```diff
@@ -76,12 +95,14 @@ export default function isElementClickable(element: Element): boolean {
    * overlapping element meaning there are no other elements blocking
    * the target element.
    *
-   * @param {Element} targetElement - The element to compare with the other
+   * @param {Element} targetElement The element to compare with the other
    * overlapping elements.
-   * @param {Element[]} overlappingElements - The overlapping elements to compare
+   * @param {Element[]} overlappingElements The overlapping elements to compare
    * against the target element.
+   * @returns {boolean} Whether the target element is the only overlapping
+   * element.
    */
-  const isOverlappingElementsMatching = (
+  const isTargetTheOnlyOverlappingElement = (
```

Comment:
Nit: the concept of "overlap" has 2+ things. If there's only one thing then it's not overlapping.

Perhaps you'll want something here like isTargetFullyViewable() or isTargetNotOverlappedByOtherElements()?

Reply:
Done